config/config-feature-flags.yaml

```diff
@@ -131,3 +131,6 @@ data:
   enable-artifacts: "false"
   # Setting this flag to "true" will enable the built-in param input validation via param enum.
   enable-param-enum: "false"
+  # Setting this flag to "true" will prevent users from creating
+  # embedded spec Taskruns or Pipelineruns.
```

Comment:
What about Steps? Now that we have StepActions, why just limit this to Tasks and Pipelines?

Reply:
Nevermind. I think including StepActions would cause UX trouble. Thats because even to do simple things, users will need to create a StepAction. For Tasks, I think that's ok.